libsplinter/src/admin/service/shared.rs

```diff
@@ -1932,12 +1932,16 @@ impl AdminServiceShared {
             return Ok(());
         }
 
-        let peer_node_pair = self.token_to_peer.get(peer_id).ok_or_else(|| {
-            AdminSharedError::ServiceProtocolError(format!(
-                "Missing service information for peer token : {}",
-                peer_id
-            ))
-        })?;
+        let peer_node_pair = match self.token_to_peer.get(peer_id) {
+            Some(peer_node_pair) => peer_node_pair,
+            None => {
+                warn!(
+                    "Ignoring connection, Missing service information for peer token {}",
```

Comment:
Maybe "Ignoring connection; missing service information for peer token: {}"

Reply:
replaced with suggested